### PR TITLE
jrglez-durham

### PR DESCRIPTION
**Pull request by jrglez**
Adds a file about Durham, UK. The file contains info about places to eat.

### CODE REVIEW

**Comment by paluchas** [comment]
Durham Cathedral is a nice place to see but is not included in the list

File: jrglez-durham.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Durham, UK
```
Comment (odimko) [inline jrglez-durham.md:1]: lovely place, I've been there many times 👍

File: jrglez-durham.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+## Places to eat
+- Lebaneat
```
Comment (odimko) [inline jrglez-durham.md]: Is it a restaurant? What's its speciality?

File: jrglez-durham.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+- Leonards Coffee House for sandwiches and :coffee:
+- Bells for fish & chips 
+
```
Comment (odimko) [inline jrglez-durham.md]: Any places to have a drink? :)

**Review by odimko — CHANGES_REQUESTED** [review]
Just a few comments.

File: jrglez-durham.md (hunk 1 above)
Comment (jrglez) [reply jrglez-durham.md:1]: I just moved here, but I like it a lot so far

**Pull request merged**